### ISSUE TITLE
New Category: Lists and New Rule: Tables must not be utilized as lists

### ISSUE THREAD

**Issue by A11YAngel** [opena11y/oaa-rules#24]
**New Category**: Lists
**New Rule**:  Tables must not be utilized as lists
**Requirement or recommendation**:  Recommended

There is no standard for the list markup available in Section 508 so this is highly recommended to ensure that the list markup is explicit to users of screen readers.

**Purpose/description**: 
Assistive technology such as screen readers do not render the list information using table markup properly. Explicit list markup must be used to convey the relationship of list items properly rather than the table markup. Use CSS to style the list structure if desired.

**Standards**:
WCAG 1.0 Priority 2
3.6 Mark up lists and list items properly
WCAG 2.0 Level A
1.3.1 Information and relationships conveyed through presentation

**Comment by jongund** [comment]
We have struggled with list related rules, maybe a more general rule that list of items should be marked up as lists.   The only problem with this type of rule it is a page level manual check, with no real indication of where there is a potential problem.   Do you have any links or examples of the problem we could look at?

**Comment by A11YAngel** [comment]
You are correct this rule is expected for the page level manual check only.
